.golangci.yml

```diff
@@ -23,6 +23,7 @@ linters:
   - unparam
   - unused
   - whitespace
+  - exhaustive
```

Comment:
Just enabling lint check, to ensure we don't forget to update `ResponseType.String` and similar switch statements without a default case